index.php

```diff
@@ -173,13 +173,20 @@ function createCountdown() {
 			if (bg != '') { newUrl += '&bg='+encodeURIComponent(bg); }
 			window.location.href = newUrl.replace(/ +/g,'+');
 		}
+		var tzValue = '';
+		function checkTzValue(input){
+			if(input.value.match(/^[+-](\d+(\.)?(\d+)?)?$/) == null && input.value != '')
+				input.value = tzValue;
+			else
+				tzValue = input.value;
```

Comment:
Code formatting:

* Space between the `if` and the open parenthesis
* Curly braces around if and else

Could you make these changes?